extensions/notebook/src/test/common/utils.test.ts

```diff
@@ -47,10 +47,16 @@ describe('Utils Tests', function () {
 
 	it('getOSPlatform', async () => {
 		should(utils.getOSPlatform()).not.throw();
+		should(utils.getOSPlatform('win32')).equal(utils.Platform.Windows);
```

Comment:
So I'm pretty strongly against this type of testing. Hardcoding platform checks in tests just to get code coverage isn't something we should be doing - instead we should be making the tests be able to run on any platform and then actually running the tests on all platforms.

Now obviously we aren't there yet for running on all platforms but that I don't see that as a good reason to do this kind of thing now.

Reply:
Okayyy